### ISSUE TITLE
IdentityHub Artefact Publishing

### ISSUE THREAD

**Issue by Izzzu** [agera-edc/MinimumViableDataspace#166]
Feature #30 
after #164 

## Description
Publish Identity Hub artifacts to central artifact repository so that it can be accessed as a dependency by other services based on the dependency management concept described in #164 .

## Acceptance criteria

- [ ] Identity Hub artifacts are publicly available and can be used as gradle dependencies
- [ ] MVD repositories use the Identity Hub published artifacts instead of publishing them locally in the workflow
- [ ] All documentation/instructions mentioning local publishing artifacts for Identity Hub is updated

**Comment by zeier** [comment]
Put on hold, as it is projected to be too much scope for the team until end of August.
#36 and #157 have higher priority.
This item can be taken up again if there is capacity.
